homeassistant/components/dynalite/bridge.py

```diff
@@ -0,0 +1,118 @@
+"""Code to handle a Dynalite bridge."""
+
+from dynalite_devices_lib import DynaliteDevices
+from dynalite_lib import CONF_ALL
+
+from homeassistant.const import CONF_HOST
+from homeassistant.core import callback
+
+from .const import DATA_CONFIGS, DOMAIN, LOGGER
+from .light import DynaliteLight
```

Comment:
We're leaking entity implementation details from the light platform to the bridge module. That's not a good pattern. The platform details should stay within the platform.

It's ok to extract a common function that's used by multiple platforms but the platform should still be responsible for creating and adding entities etc.

Reply:
fixed. will be pushed shortly